src/app/components/link-editor-popup/index.tsx

```diff
@@ -101,21 +102,20 @@ export class LinkNodeView implements NodeView {
     this.dom = document.createElement('a');
     this.dom.style.cursor = 'pointer';
     this.dom.setAttribute('href', this.node.attrs.href);
-    this.dom.addEventListener('click', this.open.bind(this));
+    this.dom.addEventListener('click', debounce(this.open.bind(this), 100));
   }
 
-  open(event?: MouseEvent) {
-    event && event.preventDefault();
-
-    const { from } = this.view.state.selection;
-    const start = this.view.coordsAtPos(from) as DOMRect;
+  open() {
+    const { $from } = this.view.state.selection;
+    console.log(this.view.state.selection);
+    const start = this.view.coordsAtPos($from.pos);
 
     this.linkEditorContainer = this.view.dom.parentNode.appendChild(document.createElement('div'));
     this.linkEditorContainer.style.position = 'absolute';
     this.linkEditorContainer.style.zIndex = '10';
 
-    const x = start.left + LINK_EDITOR_MAX_WIDTH / 2;
-    const y = start.top + start.height;
+    const x = start.left;
```

Comment:
Nothing to worry about here, but these are the cursor position values. Be interesting to see what Nick thinks, but it would in my opinion be better to anchor to the nodeView objects left value rather than the cursor. Honestly its minor though, so don't feel the need to fix this now.